iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/functionclass/ActionExecutor.java

```diff
@@ -0,0 +1,58 @@
+package com.cgi.iec61850serversimulator.functionclass;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.beanit.openiec61850.ServerModel;
+import com.beanit.openiec61850.ServerSap;
+import com.beanit.openiec61850.internal.cli.ActionException;
+import com.beanit.openiec61850.internal.cli.ActionListener;
+import com.cgi.iec61850serversimulator.dataclass.Device;
+
+/**
+ * Class which takes certain button inputs for debug features during runtime.
+ */
+
+public class ActionExecutor implements ActionListener {
+
+    private static final Logger logger = LoggerFactory.getLogger(ServerSimulator.class);
+    private static final String PRINT_SERVER_MODEL_KEY = "p";
+    private static final String DEVICE_SHOW_MODEL = "d";
+    public ServerSap serverSap = null;
+    public ServerModel serverModel = null;
+    public Device device = null;
```

Comment:
- Make serverSap and device private.
- Remote the serverModel field.

Reply:
Done